core/domain/skill_services.py

```diff
@@ -848,30 +874,35 @@ def get_skill_mastery(user_id, skill_id):
             requested.
 
     Returns:
-        degree_of_mastery: float. Mastery degree of the user for the
-            requested skill.
+        degree_of_mastery: float or None. Mastery degree of the user for the
+            requested skill, or None if UserSkillMasteryModel does not exist
+            for the skill.
     """
     model_id = user_models.UserSkillMasteryModel.construct_model_id(
         user_id, skill_id)
-    degree_of_mastery = user_models.UserSkillMasteryModel.get(
-        model_id).degree_of_mastery
+    try:
+        user_skill_mastery_model = user_models.UserSkillMasteryModel.get(
```

Comment:
I think you can just pass `strict=False` to get None if model doesn't exist.

Reply:
Done.